packages/twenty-server/src/engine/core-modules/workspace/utils/get-auth-providers-by-workspace.util.ts

```diff
@@ -21,12 +25,24 @@ export const getAuthProvidersByWorkspace = ({
       workspace.isPasswordAuthEnabled && systemEnabledProviders.password,
     microsoft:
       workspace.isMicrosoftAuthEnabled && systemEnabledProviders.microsoft,
-    sso: workspace.workspaceSSOIdentityProviders.map((identityProvider) => ({
-      id: identityProvider.id,
-      name: identityProvider.name,
-      type: identityProvider.type,
-      status: identityProvider.status,
-      issuer: identityProvider.issuer,
-    })),
+    sso: workspace.workspaceSSOIdentityProviders.reduce(
```

Comment:
not sure to understand why we reduce to make map?

Reply:
it's a map filter so a reduces because we map the field and filter on active SSO providers only.